client/asset/btc/spv.go

```diff
@@ -174,40 +177,38 @@ func (w *btcSPVWallet) Start() (SPVService, error) {
 	}
 	errCloser.Add(w.neutrinoDB.Close)
 
-	// Depending on the network, we add some addpeers or a connect peer. On
-	// regtest, if the peers haven't been explicitly set, add the simnet harness
-	// alpha node as an additional peer so we don't have to type it in. On
-	// mainet and testnet3, add a known reliable persistent peer to be used in
-	// addition to normal DNS seed-based peer discovery.
-	var addPeers []string
-	var connectPeers []string
-	switch w.chainParams.Net {
-	case wire.MainNet:
-		addPeers = []string{"cfilters.ssgen.io"}
-	case wire.TestNet3:
-		addPeers = []string{"dex-test.ssgen.io"}
-	case wire.TestNet, wire.SimNet: // plain "wire.TestNet" is regnet!
-		connectPeers = []string{"localhost:20575"}
-	}
 	w.log.Debug("Starting neutrino chain service...")
 	w.cl, err = neutrino.NewChainService(neutrino.Config{
 		DataDir:       w.dir,
 		Database:      w.neutrinoDB,
 		ChainParams:   *w.chainParams,
 		PersistToDisk: true, // keep cfilter headers on disk for efficient rescanning
-		AddPeers:      addPeers,
-		ConnectPeers:  connectPeers,
+		// AddPeers:      addPeers,
+		// ConnectPeers:  connectPeers,
 		// WARNING: PublishTransaction currently uses the entire duration
 		// because if an external bug, but even if the resolved, a typical
 		// inv/getdata round trip is ~4 seconds, so we set this so neutrino does
 		// not cancel queries too readily.
 		BroadcastTimeout: 6 * time.Second,
+		NameResolver:     net.LookupIP,
```

Comment:
This is interesting.  What is it if not specified?  We were giving it fqdns that required resolution.

Reply:
If it's not specified, it will still use net.LookupIP. I was just trying to make sure it didn't change in a future release of neutrino.

This is leftover from the previous version of the PR where I was trying to make sure that the resolver used in neutrino was the same as what is used in our code. I'll remove this since we're doing the resolving before we call `CreateNode`.